### PR TITLE
[CLD-6378]Add lambdas to data engineering

### PR DESCRIPTION
**Pull request by andrleite**
#### Summary
Add lambdas to data engineering env
#### Ticket Link
  Fixes https://mattermost.atlassian.net/browse/CLD-6327

### CODE REVIEW

**Review by angeloskyratzakos — APPROVED** [review]

**Review by stafot — APPROVED** [review]
LGTM

**Review by stafot — CHANGES_REQUESTED** [review]
Need to create `upload-data-eng-lambdas` job. Please refactore accordingly to be environment separated

File: .github/workflows/data-engineering.yml [hunk 1]
```diff
@@ -0,0 +1,15 @@
+name: Upload lambdas to S3 for data engineering
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  upload-prod-lambdas:
```
Comment (stylianosrigas) [inline .github/workflows/data-engineering.yml:9]:
```suggestion
  upload-data-engineering-lambdas:
```

**Review by stylianosrigas — CHANGES_REQUESTED** [review]
Just a minor ;)

**Comment by andrleite** [comment]
> Need to create `upload-data-eng-lambdas` job. Please refactore accordingly to be environment separated

@stafot Sorry I didn't get it. What do you mean?

**Comment by stafot** [comment]
@andrleite is the one you fixed already. Also you need to change the naming to be similar to other environments. (As `dev-ci.yaml` this should be `data-engineering-ci.yaml`

**Review by stafot — APPROVED** [review]

**Review by stylianosrigas — APPROVED** [review]

**Pull request merged**